src/traces/scatter/style.js

```diff
@@ -37,6 +43,9 @@ function style(gd, cd) {
 
 function stylePoints(sel, trace, gd) {
     Drawing.pointStyle(sel.selectAll('path.point'), trace, gd);
+}
+
+function styleText(sel, trace, gd) {
```

Comment:
Previously text was just shoved into the same group as points, which had the pleasant side-effect of letting them be styled together. Now that they're in separate groups they need independent outer selections. AFAICT none of the other users of `stylePoints` actually *wanted* to be styling text anyway (as opposed to the users of `style`), so this may be a slight 🐎 improvement... but @etpinard please take a look.

Reply:
oh hmm, I guess the image failure is telling me `geo` needs this...

Reply:
alright, put back into scatterGeo -> https://github.com/plotly/plotly.js/pull/2978/commits/7c54fc2f0bd38ecac52c314b736e86ef035c3511

Note that `scattergeo` still [removes everything on update](https://github.com/plotly/plotly.js/blob/33e1acfdd5ffb8a87a637e0868827abad60dabbf/src/traces/scattergeo/plot.js#L37) so these ordering bugs should not matter there, but we should be able to use essentially the same structure there as here to switch to a more standard update pattern without introducing similar bugs.

Reply:
> alright, put back into scatterGeo -> 7c54fc2

Glad to see our tests caught that. 

Now that only scattergeo uses it, maybe we could move `styleText` to 

https://github.com/plotly/plotly.js/blob/master/src/traces/scattergeo/style.js ?

Reply:
Ah never mind, `styleText` is used in `styleOnSelect`.

Reply:
> AFAICT none of the other users of stylePoints actually wanted to be styling text anyway (as opposed to the users of style), so this may be a slight  improvement... but @etpinard please take a look.

:+1: for having text and points in different groups!